projects/1.16.1/assets/crossroads-mc/crossroads/lang/zh_cn.json

```
@@ -335,7 +336,7 @@
 	"tt.crossroads.heat_battery.info": "作为缓存减少温度变化",
```

```suggestion
	"tt.crossroads.heat_battery.info": "行为类似于缓存，可以减少温度变化",
```
原句一下子塞了三个实义句子成分，所以可以考虑把句子拆分，中文在句子内表述过多实义成分的时候会让阅读者的阅读难度直线上升，所以遇到这种情况可以拆句。比较典型的就是英语句子中的定语部分拆分，一个是定语从句，另一个是非谓语动词做定语，这些定语往往非常长，有时甚至相当复杂，这时候肯定是需要把这些定语拆分出来，单独成一个小句子。